handlers/guest_links.go

```diff
@@ -63,6 +63,40 @@ func (s Server) guestLinksDelete() http.HandlerFunc {
 	}
 }
 
+func (s *Server) guestLinksDisable() http.HandlerFunc {
```

Comment:
Can we add unit tests for this? See `TestDeleteExistingGuestLink` as an example.

(ditto for enable)